engines/cryo/detection.cpp

```diff
@@ -133,15 +133,18 @@ static const ADGameDescription gameDescriptions[] = {
 class CryoMetaEngine : public AdvancedMetaEngine {
 public:
 	CryoMetaEngine() : AdvancedMetaEngine(Cryo::gameDescriptions, sizeof(ADGameDescription), cryoGames) {
-		_singleId = "losteden";
+	}
+
+	const char *getEngineId() const {
+		return "cryo";
 	}
 
 	virtual const char *getName() const {
-		return "Cryo Engine";
+		return "Cryo";
 	}
 
 	virtual const char *getOriginalCopyright() const {
-		return "Cryo Engine (C) Cryo Interactive";
+		return "Cryo (C) Cryo Interactive";
```

Comment:
That makes no sense. It should stay "Cryo Engine". Or, if you want to put the game name, then it is "Lost Eden".

Reply:
Reverted